internal/third_party/kubernetes/flexvolume/flexvolume.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package flexvolume is copied from k8s.io/kubernetes/pkg/volume/flexvolume.
+package flexvolume
+
+// Source: https://github.com/kubernetes/kubernetes/blob/v1.21.1/pkg/volume/flexvolume/driver-call.go
```

Comment:
The flex driver will hopefully be going away soon anyway with #8076, but the timeline hasn't been finalized yet.